boot/zephyr/CMakeLists.txt

```diff
@@ -252,6 +252,26 @@ if(CONFIG_MCUBOOT_SERIAL)
     )
 endif()
 
+if(CONFIG_SOC_ESP32 OR CONFIG_SOC_ESP32S2 OR CONFIG_SOC_ESP32C3 OR CONFIG_SOC_ESP32S3)
```

Comment:
We plan to have some `CONFIG_FAMILY_SOC_ESP32` soon. For now, this works.

Reply:
I added `CONFIG_FAMILY_SOC_ESP32` into related Zephyr PR#54835